app/views/search/_form.html.erb

```diff
@@ -0,0 +1,83 @@
+<%
+# Conditional input label depends on whether a search has already happened.
```

Comment:
This setup block feels like good candidates for a couple of well-named helpers. I'm okay with leaving this here for now though.